### PR TITLE
Move slider configuration.

### PR DESCRIPTION
**Pull request by bierik**
Expose configuration for slider in DOM as a data attribute.

### CODE REVIEW

**Comment by mbaechtold** [comment]
@bierik: #4 fixes the buildout. Once #4 has been merged, you may rebase your branch onto master.

**Comment by mbaechtold** [comment]
@bierik: #4 has been merged.

**Comment by bierik** [comment]
@maethu rebased

File: ftw/sliderblock/browser/sliderblock.py [hunk 1]
```diff
@@ -9,8 +9,5 @@ def get_panes(self):
         return self.context.getFolderContents(full_objects=True)
 
     def get_slick_config(self):
-        js = "jQuery(function($) {{" \
-             "  $('#uid_{uid}.sliderWrapper .sliderPanes').slick({config});" \
-             "}});"
-        return js.format(uid=self.context.UID(),
-                         config=self.context.slick_config)
+        return "{config}".format(uid=self.context.UID(),
```
Comment (maethu) [inline ftw/sliderblock/browser/sliderblock.py]: `return self.context.slick_config`

**Comment by bierik** [comment]
@maethu updated

**Pull request merged**